prac_08/dynamic_labels.py

```diff
@@ -0,0 +1,30 @@
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.uix.button import Button
+from kivy.properties import StringProperty
+
+
+class DynamicLabelsApp(App):
+    status_text = StringProperty()
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.names = ["john", "michelle", "tommy", "charlie"]
+
+    def build(self):
+        self.title = "Dynamic Label"
+        self.root = Builder.load_file('dynamic_labels.kv')
+        self.create_widgets()
+        return self.root
+
+    def create_widgets(self):
+        for name in self.names:
+            temp_button = Button(text=name)
```

Comment:
Great looking code overall! 
On this part, I think we needed to create Labels?

from [Prac 8 Dynamic Labels](https://github.com/CP1404/Practicals/tree/master/prac_08)
"Your app won't have any buttons or interactivity."